locales/app/zh-HK.json

```
@@ -122,59 +122,59 @@
   "entry_list_header.items": "項目",
   "entry_list_header.new_entries_available": "有新條目可用",
   "entry_list_header.refetch": "重新抓取",
-  "entry_list_header.refresh": "刷新",
+  "entry_list_header.refresh": "重新整理",
   "entry_list_header.show_all": "顯示所有",
   "entry_list_header.show_all_items": "顯示所有條目",
   "entry_list_header.show_unread_only": "僅顯示未讀",
   "entry_list_header.switch_to_grid": "切換到網格視圖",
   "entry_list_header.switch_to_masonry": "切換到 Masonry 視圖",
   "entry_list_header.switch_to_normalmode": "切換至正常模式",
-  "entry_list_header.switch_to_widemode": "切換至寬屏模式",
+  "entry_list_header.switch_to_widemode": "切換至寬螢幕模式",
   "entry_list_header.unread": "未讀",
-  "feed.follower_one": "關注者",
-  "feed.follower_other": "關注者",
+  "feed.follower_one": "追隨者",
+  "feed.follower_other": "追隨者",
   "feed.followsAndFeeds": "在 {{appName}} 上有 {{subscriptionCount}} 個 {{subscriptionNoun}} 和 {{feedsCount}} 個 {{feedsNoun}}",
   "feed.followsAndReads": "在 {{appName}} 上有 {{subscriptionCount}} 個 {{subscriptionNoun}} 和 {{readCount}} 個 {{readNoun}}",
-  "feed.read_one": "閲讀",
-  "feed.read_other": "閲讀",
+  "feed.read_one": "閱讀",
+  "feed.read_other": "閱讀",
   "feed_claim_modal.choose_verification_method": "有三種驗證方法可選，你可以選擇其中一種進行驗證。",
   "feed_claim_modal.claim_button": "認領",
   "feed_claim_modal.content_instructions": "複製以下內容並貼到你最新的 RSS 訂閱源中",
   "feed_claim_modal.description_current": "當前描述：",
   "feed_claim_modal.description_instructions": "複製以下內容並貼到你的 RSS 訂閱源中的 <code /> 欄位",
-  "feed_claim_modal.failed_to_load": "無法加載認領信息",
+  "feed_claim_modal.failed_to_load": "無法載入認領資訊",
   "feed_claim_modal.rss_format_choice": "RSS 生成器通常有兩種格式可選。請根據需要複製以下的 XML 和 JSON 格式。",
-  "feed_claim_modal.rss_instructions": "複製以下代碼並貼到你的 RSS 生成器中",
+  "feed_claim_modal.rss_instructions": "複製以下編號並貼到你的 RSS 生成器中",
   "feed_claim_modal.rss_json_format": "JSON 格式",
   "feed_claim_modal.rss_xml_format": "XML 格式",
-  "feed_claim_modal.rsshub_notice": "此訂閱源由 RSSHub 提供，緩存時間為 1 小時。發布內容後，請等待最多 1 小時更改才能生效。",
+  "feed_claim_modal.rsshub_notice": "此訂閱源由 RSSHub 提供，緩存時間為 1 小時。發佈內容後，請等待最多 1 小時更改才能生效。",
```

```suggestion
  "feed_claim_modal.rsshub_notice": "此訂閱源由 RSSHub 提供，緩存時間為 1 小時。發布內容後，請等待最多 1 小時更改才能生效。",
```
Reference: https://www.isd.gov.hk/chi/news_agency.htm